jujugui/static/gui/src/app/components/terminal/_terminal.scss

```diff
@@ -1,17 +1,58 @@
-.juju-shell__terminal-container {
+.juju-shell {
   width: 100%;
-  height: 100%;
-  margin: 0 auto;
-  padding: 2px;
+  background-color: #111;
 
+  &__header {
+    background-color: $dark-warm-grey;
+    color: $light-grey;
+    text-align: center;
+    line-height: 30px;
+    height: 30px;
+    position: relative;
+
+    &-actions {
+      position: absolute;
+      top: 0;
+      right: 0;
+      line-height: 36px;
+
+      .svg-icon {
+        stroke: #cdcdcd;
+        fill: #cdcdcd;
```

Comment:
This can be `$mid-grey`

Reply:
I can't seem to find this being defined anywhere?

Reply:
These colours are defined in Vanilla: https://github.com/vanilla-framework/vanilla-framework/blob/v0.1.0/scss/_global-settings.scss